frontend/src/components/Tag/Tag.tsx

```diff
@@ -39,7 +39,9 @@ export const TagRightIcon = forwardRef<TagIconProps, 'svg'>((props, ref) => {
   )
 })
 
-const TagCloseIcon = () => <Icon as={BiX} fontSize="1.5rem" />
+const TagCloseIcon = () => (
+  <Icon as={BiX} fontSize="1.5rem" aria-label="Remove option icon" />
```

Comment:
misnomer imho. should we use  'tag' instead, which is more generic. maybe we can allow prop overrides too

Reply:
The reason for using "option" is that in the close button component (right below this), `aria-label="Remove selected option"` is used hahah

Reply:
wondering also if "tag" is common terminology for this type of usecase, maybe it should actually read "Remove <tag value> icon"

Reply:
actually ya hor, tag probably less obvious than option. option it is!